x-pack/plugins/apm/server/lib/helpers/get_ml_setup.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isActivePlatinumLicense } from '../../../common/license_check';
+import { APMRouteHandlerResources } from '../../routes/typings';
+
+export interface MlSetup {
+  mlSystem: any;
+  anomalyDetectors: any;
+  modules: any;
+}
+
+export async function getMlSetup({
```

Comment:
Let's call it "clients" for consistency:

```suggestion
export async function getMlClients({
```

Reply:
Clients? in plural?

Reply:
No, probably `getMlClient` is better. I think I used plural because it returns multiple providers.